2022-08-31 02:25:07,106 INFO    MainThread:95502 [wandb_setup.py:_flush():76] Configure stats pid to 95502
2022-08-31 02:25:07,106 INFO    MainThread:95502 [wandb_setup.py:_flush():76] Loading settings from /mmfs1/home/tjung2/.config/wandb/settings
2022-08-31 02:25:07,106 INFO    MainThread:95502 [wandb_setup.py:_flush():76] Loading settings from /mmfs1/gscratch/ark/tjung2/continual-knowledge-learning/wandb/settings
2022-08-31 02:25:07,108 INFO    MainThread:95502 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-08-31 02:25:07,108 INFO    MainThread:95502 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'run.py', 'program': 'run.py'}
2022-08-31 02:25:07,108 INFO    MainThread:95502 [wandb_init.py:_log_setup():461] Logging user logs to /mmfs1/gscratch/ark/tjung2/continual-knowledge-learning/wandb/run-20220831_022506-1a7igc0b/logs/debug.log
2022-08-31 02:25:07,108 INFO    MainThread:95502 [wandb_init.py:_log_setup():462] Logging internal logs to /mmfs1/gscratch/ark/tjung2/continual-knowledge-learning/wandb/run-20220831_022506-1a7igc0b/logs/debug-internal.log
2022-08-31 02:25:07,108 INFO    MainThread:95502 [wandb_init.py:init():495] calling init triggers
2022-08-31 02:25:07,109 INFO    MainThread:95502 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {'output_dir': 'outputs/kadapters_2010_2freeze', 'dataset': 'templama', 'dataset_version': '2010', 'prefix': True, 'split_num': 1, 'split': 0, 'model_name_or_path': 'google/t5-small-ssm', 'method': 'kadapter', 'freeze_level': 2, 'mode': 'pretrain', 'tokenizer_name_or_path': 'google/t5-small-ssm', 'max_input_length': 50, 'max_output_length': 25, 'freeze_encoder': False, 'freeze_embeds': False, 'learning_rate': 0.001, 'weight_decay': 0.0, 'adam_epsilon': 1e-08, 'warmup_steps': 0, 'train_batch_size': 32, 'eval_batch_size': 32, 'num_train_epochs': 90, 'n_gpu': 1, 'num_workers': 4, 'resume_from_checkpoint': None, 'use_lr_scheduling': True, 'val_check_interval': 1.0, 'n_val': -1, 'n_train': -1, 'n_test': -1, 'early_stop_callback': False, 'use_deepspeed': False, 'opt_level': 'O1', 'max_grad_norm': 0.5, 'seed': 42, 'check_validation_only': False, 'checkpoint_path': '', 'accelerator': 'ddp', 'output_log': None, 'wandb_log': True, 'adapter_list': [1, 5, 8], 'adapter_hidden_size': 128, 't5_learning_rate': None, 'checkpoint_dir': None, 'adapter_enc_dec': None}
2022-08-31 02:25:07,109 INFO    MainThread:95502 [wandb_init.py:init():548] starting backend
2022-08-31 02:25:07,109 INFO    MainThread:95502 [wandb_init.py:init():552] setting up manager
2022-08-31 02:25:07,110 INFO    MainThread:95502 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-31 02:25:07,116 INFO    MainThread:95502 [wandb_init.py:init():558] backend started and connected
2022-08-31 02:25:07,121 INFO    MainThread:95502 [wandb_init.py:init():636] updated telemetry
2022-08-31 02:25:07,172 INFO    MainThread:95502 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2022-08-31 02:25:37,173 INFO    MainThread:95502 [interface_shared.py:_communicate_run():331] couldn't get run from backend
2022-08-31 02:25:37,174 ERROR   MainThread:95502 [wandb_init.py:init():676] backend process timed out
2022-08-31 02:25:37,174 ERROR   MainThread:95502 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-08-31 02:26:06,447 WARNING MsgRouterThr:95502 [router.py:message_loop():76] message_loop has been closed
